libmscore/sym.cpp

```diff
@@ -2842,6 +2844,8 @@ const std::array<const char*, int(SymId::lastSym)+1> Sym::symNames = { {
       "windTrillKey",
       "windVeryTightEmbouchure",
       "windWeakAirPressure",
+      // SMuFL standard symbol IDs }}}
+
 
```

Comment:
one empty line too much for my taste

Reply:
The empty line makes it easier to see where the SMuFL-derived stuff begins and ends. (Yes, we also have the comment, but…)

If you insist, I can remove it.

Reply:
Oh wait, I see. Sorry. Fixed.